components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/security/usermgt/APIKeyMgtRemoteUserClientPool.java

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
invalid licence header

Reply:
Not applicable anymore since the master branch has a different codebase.